### PR TITLE
BA-324 create 50px spacer custom block

### PR DESCRIPTION
**Pull request by lisafolkerson**
Create a custom component to standardize spacing between components

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -44,6 +44,11 @@
     "sort-packages": true,
     "platform": {
       "php": "7.3"
+    },
+    "allow-plugins": {
```
Comment (Braedencraig) [inline composer.json:48]: Do we need these added 100%? Assuming you needed these to get your local up and running appropriately? They are fine to stay in im sure!

**Review by Braedencraig — APPROVED** [review]

**Comment by Braedencraig** [comment]
@lisafolkerson also this is a merge to master! WE wanna pop this into develop first!

**Review by Braedencraig — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (lisafolkerson) [reply composer.json:48]: Yeah these came along with having to update lando and composer so i think we're just a bit fresher with our versions and deps now!

Comment (Braedencraig) [reply composer.json:48]: heck yes

**Pull request merged**